samples/word/set_the_font_for_a_text_run/cs/Program.cs

```diff
@@ -18,6 +18,11 @@ static void SetRunFont(string fileName)
                 Ascii = "Arial"
             });
 
+        if (package.MainDocumentPart is null)
+        {
+            throw new System.NullReferenceException("MainDocumentPart is null.");
```

Comment:
These should be removed and replaced with ArgumentNullException or InvalidOperationException throughout these changes